pytorch_toolkit/action_recognition/main.py

```diff
@@ -352,7 +352,12 @@ def main():
             class_weights = getattr(args, 'class_weights', None)
             sampler = WeightedRandomSampler(train_data.get_sample_weights(class_weights), len(train_data))
         else:
-            sampler = RandomSampler(train_data)
+            replacement = False
+            num_samples = len(train_data)
+            if len(train_data) < args.batch_size:
+                replacement = True
+                num_samples = args.batch_size
+            sampler = RandomSampler(train_data, replacement=replacement, num_samples=num_samples)
```

Comment:
sampler = RandomSampler(train_data, replacement=replacement, num_samples=num_samples)
  File "/home/user/work/PR/ar/pytorch_toolkit/action_recognition/venv/lib/python3.6/site-packages/torch/utils/data/sampler.py", line 89, in __init__
    raise ValueError("With replacement=False, num_samples should not be specified, "
ValueError: With replacement=False, num_samples should not be specified, since a random permute will be performed.

Reply:
Which version of PyTorch do use for testing?

Reply:
1.4.0

Reply:
Should be fixed now.